cmd/root.go

```diff
@@ -13,8 +13,9 @@ var cfgFile string
 
 // rootCmd represents the base command when called without any subcommands
 var rootCmd = &cobra.Command{
-	Use:   "truss-cli",
-	Short: "A CLI for use with Bridge Truss",
+	Use:     "truss-cli",
+	Short:   "A CLI for use with Bridge Truss",
+	Version: "v0.0.6",
```

Comment:
I think it's important to have this for debugging and such.

Any ideas on how to add this into the release pipeline so we don't forget to increment it? I suppose the makefile script could set this and add a git commit before triggering the release. Does that see reasonable?

Reply:
randy has done some work on this with muss

Reply:
Ahh nice, I'll take a look at that